widgets-samples/cc/samples-cc-react-app/src/App.tsx

```diff
@@ -587,6 +589,10 @@ function App() {
                 </Button>
               </div>
             )}
+
+            {isSdkReady && (store.isAgentLoggedIn || isLoggedIn) && (
+              <EngageWidget accessToken={accessToken} currentTheme={currentTheme} isSdkReady={isSdkReady} />
+            )}
```

Comment:
Since this is a separate widget anyway, can we add a bottom left floating chat window kind of thing that pops up on click of a Task? That way, the users will know that it came through. Just to make it look a bit more convenient